applications/StructuralMechanicsApplication/tests/beam_test/semi_rigid_linear_3D2NBeamCr_test_material.json

```diff
@@ -0,0 +1,25 @@
+{
+    "properties" : [{
+        "model_part_name" : "Structure.Parts_structure",
+        "properties_id"   : 1,
+        "Material"        : {
+            "constitutive_law" : {
+                "name" : "BeamConstitutiveLaw"
+            },
+            "Variables"        : {
+                "DENSITY"           : 7850.0,
+                "YOUNG_MODULUS"     : 210000000000.0,
+                "POISSON_RATIO"     : 0.30,
+                "CROSS_AREA"        : 0.00001,
+                "TORSIONAL_INERTIA" : 1e-5,
+                "I22"               : 1e-5,
+                "I33"               : 1e-5,
+                "SEMI_RIGID_ROTATIONAL_STIFFNESS_VECTOR_AXIS_2": [0e0],
+                "SEMI_RIGID_ROTATIONAL_STIFFNESS_VECTOR_AXIS_3": [1e7],
+                "SEMI_RIGID_NODE_IDS": [6]
```

Comment:
I am pretty hesitant using IDs as input because they are reordered in MPI

How about using Nodal-Data to specify this?